cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/AbstractQueueAdmin.java

```diff
@@ -78,38 +57,32 @@ public static String getFlowName(String queueTableName) {
     return parts[parts.length - 1];
   }
 
+  public static  TableId getConfigTableId(QueueName queueName) {
```

Comment:
I think only HBase implementation has config table. Why it's needed in AbstractQueueAdmin?